worker/deploy_device_configuration.json

```
@@ -22,6 +22,20 @@
                 "connectionTimeOut": 8000,
                 "readTimeOut": 8000
             }
+        },
+        {
+            "name": "trigger_check_update",
+            "type": "http",
+            "retries": 3,
+            "http": {
+                "uri": "http://${env.DEVICECONNECT_ADDR|mender-deviceconnect:8080}/api/internal/v1/deviceconnect/tenants/${workflow.input.tenant_id}/devices/${workflow.input.device_id}/check-update",
+                "method": "POST",
+                "headers": {
+                    "X-MEN-RequestID": "${workflow.input.request_id}"
+                },
+                "connectionTimeOut": 8000,
+                "readTimeOut": 8000
```

The device could be non-provisioned in deviceconnect (404), or offline (409). In these cases, the workflow will fail. I think it makes sense to consider these status codes as valid, in the context of this workflow.

```suggestion
                "readTimeOut": 8000,
                "statusCodes": [202, 404, 409],
```

API documentation: https://github.com/mendersoftware/deviceconnect/blob/master/docs/internal_api.yml#L153